lib/src/property.dart

```diff
@@ -278,10 +278,11 @@ class Color implements _StyleProperty, ColorBase {
           return new Hsla(args[0], args[1], args[2], args[3]).toHexArgbString();
         default:
           // Type not defined UnsupportedOperationException should have thrown.
-          assert(true);
+          assert(false);
```

Comment:
Ah, neither ... there is a message about missing return, and the John Analyzer noticed it :)